From 01fdee939a81bb17e93ad026c3eb496e9c5db31f Mon Sep 17 00:00:00 2001
From: Liviu Rau <liviurau@chromium.org>
Date: Tue, 19 Jan 2021 17:39:56 +0100
Subject: [PATCH] [branches] Cleanup mb_config

Bug: chromium:1064551
Change-Id: I81935d9bfe7b779f6f33ae8afa54f5b86c6bd74e
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/2637224
Reviewed-by: Michael Achenbach <machenbach@chromium.org>
Commit-Queue: Liviu Rau <liviurau@chromium.org>
Cr-Commit-Position: refs/heads/master@{#72184}
---
 infra/mb/mb_config.pyl | 34 ----------------------------------
 1 file changed, 34 deletions(-)

diff --git a/infra/mb/mb_config.pyl b/infra/mb/mb_config.pyl
index 9b2b60b11e..6a23542a74 100644
--- a/infra/mb/mb_config.pyl
+++ b/infra/mb/mb_config.pyl
@@ -175,38 +175,6 @@
       # RISC-V
       'V8 Linux - riscv64 - sim - builder': 'release_simulate_riscv64',
     },
-    'client.v8.branches': {
-      'V8 Linux - previous branch': 'release_x86',
-      'V8 Linux - previous branch - debug': 'debug_x86',
-      'V8 Linux - beta branch': 'release_x86',
-      'V8 Linux - beta branch - debug': 'debug_x86',
-      'V8 Linux - stable branch': 'release_x86',
-      'V8 Linux - stable branch - debug': 'debug_x86',
-      'V8 Linux64 - previous branch': 'release_x64',
-      'V8 Linux64 - previous branch - debug': 'debug_x64',
-      'V8 Linux64 - beta branch': 'release_x64',
-      'V8 Linux64 - beta branch - debug': 'debug_x64',
-      'V8 Linux64 - stable branch': 'release_x64',
-      'V8 Linux64 - stable branch - debug': 'debug_x64',
-      'V8 arm - sim - previous branch': 'release_simulate_arm',
-      'V8 arm - sim - previous branch - debug': 'debug_simulate_arm',
-      'V8 arm - sim - beta branch': 'release_simulate_arm',
-      'V8 arm - sim - beta branch - debug': 'debug_simulate_arm',
-      'V8 arm - sim - stable branch': 'release_simulate_arm',
-      'V8 arm - sim - stable branch - debug': 'debug_simulate_arm',
-      'V8 mips64el - sim - previous branch': 'release_simulate_mips64el',
-      'V8 mips64el - sim - beta branch': 'release_simulate_mips64el',
-      'V8 mips64el - sim - stable branch': 'release_simulate_mips64el',
-      'V8 mipsel - sim - previous branch': 'release_simulate_mipsel',
-      'V8 mipsel - sim - beta branch': 'release_simulate_mipsel',
-      'V8 mipsel - sim - stable branch': 'release_simulate_mipsel',
-      'V8 ppc64 - sim - previous branch': 'release_simulate_ppc64',
-      'V8 ppc64 - sim - beta branch': 'release_simulate_ppc64',
-      'V8 ppc64 - sim - stable branch': 'release_simulate_ppc64',
-      'V8 s390x - sim - previous branch': 'release_simulate_s390x',
-      'V8 s390x - sim - beta branch': 'release_simulate_s390x',
-      'V8 s390x - sim - stable branch': 'release_simulate_s390x',
-    },
     'tryserver.v8': {
       'v8_android_arm_compile_rel': 'release_android_arm',
       'v8_android_arm64_compile_dbg': 'debug_android_arm64',
@@ -564,8 +532,6 @@
       'debug', 'x86', 'goma', 'v8_enable_slow_dchecks', 'v8_full_debug'],
 
     # Release configs for x86.
-    'release_x86': [
-      'release_bot', 'x86'],
     'release_x86_gcc': [
       'release_bot_no_goma', 'x86', 'gcc', 'v8_check_header_includes'],
     'release_x86_gcc_minimal_symbols': [
-- 
2.35.1

